dist/statful.js

```diff
@@ -127,21 +159,14 @@
             Object.keys(typeTags).forEach(function(key) {
                 tags[key] = typeTags[key];
             });
-            if (!methodTags) {
-                tags = tags || {};
-                if (environment) {
-                    tags.env = environment;
-                }
-            } else {
-                Object.keys(methodTags).forEach(function(key) {
-                    tags[key] = methodTags[key];
-                });
-                if (!tags.env && environment) {
-                    tags.env = environment;
-                }
-                if (!tags.app && app) {
-                    tags.app = app;
-                }
+            Object.keys(methodTags).forEach(function(key) {
+                tags[key] = methodTags[key];
+            });
+            if (!tags.env && environment) {
+                tags.env = environment;
```

Comment:
I'm wondering if we want to set this default tag, I don't think `environment` should be a config attribute. IIRC this isn't on the spec either although it was on the previous versions of the client.

I'd say If the user wants to add an environment tag then he can do it by setting a default global tag himself.